packages/redux/src/store.ts

```diff
@@ -57,6 +57,8 @@ export interface WorkspaceSettings {
   defaultNotebookSettings: {
     isMinimapEnabled?: boolean;
   };
+  webgl: boolean;
```

Comment:
Will these always be defined?

Reply:
Yes. See in the `getSettings` selector, we start from all the default settings, and override the ones the user has set. The users settings are a partial: https://github.com/deephaven/web-client-ui/blob/232fb4db121cd507d0303dba6b468ae1edab9135/packages/redux/src/selectors.ts#L71